packages/good-design/src/core/layout/ClaimCarousel.tsx

```diff
@@ -0,0 +1,98 @@
+import { FlatList, View } from "native-base";
+import React, { FC, useCallback, useMemo, useState } from "react";
+import { LayoutChangeEvent, NativeScrollEvent, NativeSyntheticEvent } from "react-native";
+import { IClaimCard } from "../buttons";
+import ClaimCard from "./ClaimCard";
+
+interface ClaimCarouselProps {
+  cards: Array<IClaimCard>;
+}
+
+const ClaimCarousel: FC<ClaimCarouselProps> = ({ cards }) => {
```

Comment:
there's ready solution for carousel, have you checked it ? or tried to find another one ?
https://stackoverflow.com/questions/41372423/react-native-nativebase-content-component-and-carousel-not-displaying

Reply:
tried several most popular, non of them didn't work as it needed. Also sliding is a built-in feature in FlatList, so no need to try some libraries